### DISCUSSION TITLE
Propertising flymake output in Eldoc

### DISCUSSION THREAD

**Discussion by trev-dev** [joaotavora/eglot#891] [Q&A]
Thanks to Joao I was able to debug my eldoc config and get my flymake messages to show up along side my documentation strings.

I noticed that these diagnostic messages have no properties :) I was wondering if anyone was aware of a simple way to hook into the flymake eldoc functions and apply faces to them. Ideally italic + the color of their urgency. Not required, but would make them pop for me.

**Comment by joaotavora** [comment]
That's an interesting feature request for Flymake itself.  See `flymake-eldoc-function`. I don't have time to do it now.  Maybe you can try it or someone else can?

**Reply by trev-dev** [reply]
II will start there! With any luck I might have something worthy of note or perhaps a pull request.

**Comment by joaotavora** [comment]
It's in Emacs itself so it'll probably be a simple patch, not a PR

**Reply by trev-dev** [reply]
I see! I have never attempted to contribute to Emacs directly. This will be (another) learning experience.
